python/lsst/pipe/base/argumentParser.py

```diff
@@ -732,19 +724,28 @@ def __call__(self, parser, namespace, values, option_string):
         @param[in] parser       argument parser (instance of ArgumentParser)
         @param[in] namespace    parsed command (an instance of argparse.Namespace); ignored
         @param[in] values       a list of trace levels;
-            each item must be of the form component_name=level
+            each item must be of the form component_name=level or level
```

Comment:
Put quotes around the form, to distinguish them.  And I think it would help to put 'level' first.

    each item must be of the form 'level' or 'component_name=level'